src/opustrainer/logger.py

```diff
@@ -62,4 +62,4 @@ def setup_logger(outputfilename: Optional[str] = None, loglevel: str = "INFO", d
     # even if it has already been shutdown. This is why we use force=True to force recreation of logger so we can
     # properly run our tests. Not the best solution, not sure if it's not prone to race conditions, but it is
     # at the very least safe to use for the actual software running
-    logging.basicConfig(handlers=handlers, level=get_log_level(loglevel), format=loggingformat, datefmt='%Y-%m-%d %H:%M:%S', force=True, **kwargs)
```

Comment:
I don't understand the purpose of `**kwargs` here?

Reply:
It's because you added `encoding='utf-8'` but the encoding argument is only available since Python 3.9. So a couple of lines above this one I only set kwargs to included the encoding bit if Python is 3.9 or higher.

Reply:
oh, right.. Will fix. it